additional data field (#2)
Innim/flutter_external_login_interface: CHANGELOG.md

## [0.1.7] - 2024-05-23

> * Add field `data` from ExternalLoginResult for more information on authorisation.

Reviewer: ```suggestion
* Add field `data` to `ExternalLoginResult` for any additional information.
```
